Description: Make a path for GTK settings if it does not exist.
Author: P.L. Lucas <selairi@gmail.com>
Bug: https://github.com/lxqt/lxqt/issues/1548
Applied-Upstream: https://github.com/lxqt/lxqt-config/commit/5fe0f9288e432a669686af14657b7f508eefb77e
Last-Update: 2018-09-03
--- a/lxqt-config-appearance/configothertoolkits.cpp
+++ b/lxqt-config-appearance/configothertoolkits.cpp
@@ -212,6 +212,10 @@ void ConfigOtherToolKits::writeConfig(QS
     path = _get_config_path(path);
 
     QFile file(path);
+    if(! file.exists()) {
+        QFileInfo fileInfo(file);
+        QDir::home().mkpath(fileInfo.path());
+    }
     if (!file.open(QIODevice::WriteOnly | QIODevice::Text)) {
         return;
     }
